jwk/migrations/sql/tests/cockroach/1_test.sql

```diff
@@ -0,0 +1,4 @@
+-- +migrate Up
```

Comment:
Same here:

```
INSERT INTO hydra_jwk (sid, kid, version, keydata, created_at) VALUES ('4-sid', '4-kid', 0, 'some-key', NOW());
```

Reply:
This has the latest contents except for the `4-` to `1-`.